vm-setup/roles/v1aX_integration_test/vars/main.yml

```diff
@@ -2,7 +2,8 @@
 # Ansible playbook related variables
 HOME: "{{ lookup('env', 'HOME') }}"
 NAMESPACE: "{{ lookup('env', 'NAMESPACE') | default('metal3', true) }}"
-IRONIC_NAMESPACE: "{{ lookup('env', 'IRONIC_NAMESPACE') | default('capm3-system', true) }}"
+IRONIC_NAMESPACE: "{{ lookup('env', 'IRONIC_NAMESPACE') | default('baremetal-operator-system', true) }}"
```

Comment:
what if the test is running on v1alpha4?

Reply:
Setting the namespace for different api versions is handled [here](https://github.com/metal3-io/metal3-dev-env/blob/5bac171b55a60e9468e1fd7739b7499bd2fa591f/lib/common.sh#L170).